src/measurement.rs

```diff
@@ -80,25 +83,33 @@ impl DerefMut for Timestamp {
 impl Serialize for Timestamp {
     /// Serialize an Instant as a javascript timestamp (f64 containing the number
     /// of milliseconds since Unix Epoch 0).
-    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    fn serialize<S>(&self, serializer: S) -> std::result::Result<S::Ok, S::Error>
     where
         S: Serializer,
     {
-        let age_as_sys = self.in_system_time();
-        let timestamp = age_as_sys.duration_since(SystemTime::UNIX_EPOCH).unwrap();
-        let js_timestamp = 1000.0 * timestamp.as_secs_f64();
-        js_timestamp.serialize(serializer)
+        use serde::ser::Error;
+        match {
+            || {
+                let time = self.in_system_time()?;
+                let timestamp = time.duration_since(SystemTime::UNIX_EPOCH)?;
+                let js_timestamp = 1000.0 * timestamp.as_secs_f64();
+                anyhow::Ok(js_timestamp)
+            }
+        }() {
```

Comment:
Well that took me a bit to figure out. We should find a solution my brain can parse in a matter of seconds, not minutes.

How about adding a `fn as_js_timestamp(&self) -> Result<f64>` method to `Timestamp` and using it here?.